addon/models/registration.js

```diff
@@ -30,7 +30,11 @@ export default Node.extend({
         inverse: null
     }),
     contributors: DS.hasMany('contributors'),
-    comments: DS.hasMany('comments'),
+    comments: DS.hasMany('comments', {
+        updateRequest: {
+            requestType: () => 'POST'
```

Comment:
With newest `osf-adapter`, custom options to the relationship field (`updateRequest`) are no longer used.